pkg/injector/webhook.go

```diff
@@ -325,9 +325,9 @@ func (wh *webhook) mustInject(pod *corev1.Pod, namespace string) (bool, error) {
 	return false, nil
 }
 
-func isAnnotatedForInjection(annotations map[string]string) (exists bool, enabled bool, err error) {
+func isAnnotatedForInjection(annotations map[string]string, objectKind string, objectName string) (exists bool, enabled bool, err error) {
 	inject := strings.ToLower(annotations[constants.SidecarInjectionAnnotation])
-	log.Trace().Msgf("Sidecar injection annotation: '%s:%s'", constants.SidecarInjectionAnnotation, inject)
+	log.Trace().Msgf("%s %s has sidecar injection annotation: '%s:%s'", objectKind, objectName, constants.SidecarInjectionAnnotation, inject)
```

Comment:
I don't know if it matters in terms of simplification, but we can get rid of this trace log. It isn't necessary given we have unit tests and log when this function returns an error.